test/cql-pytest/test_describe.py

```diff
@@ -500,6 +501,13 @@ def test_desc_udf_uda(cql, test_keyspace):
             with pytest.raises(InvalidRequest):
                 cql.execute(f"DESC AGGREGATE {test_keyspace}.{fn}")
 
+def test_whitespaces_in_table_options(cql, test_keyspace):
```

Comment:
Please add a comment what the this is testing - i.e., that DESC TABLE used to print `{'enabled':true}` instead of `{'enabled': true}`.  The regex is almost impossible to understand so a reader needs to work very hard to understand why this test exists. Please also mention that this test reproduces issue #????. It's very important when you see a test failing and don't know why this test was written, or what fix you might have forgotten to backport.
By the way, why do you even need a regex in this test, if you are looking for `enabled': true` in the output, why not check this explicitly, i.e, `"'enabled': true" in desc or something like that?